flixel/system/frontEnds/SoundFrontEnd.hx

```diff
@@ -112,17 +114,20 @@ class SoundFrontEnd
 	 * @param	Volume			How loud to play it (0 to 1).
 	 * @param	Looped			Whether to loop this sound.
 	 * @param	Group			The group to add this sound to.
-	 * @param	AutoDestroy		Whether to destroy this sound when it finishes playing.  Leave this value set to "false" if you want to re-use this FlxSound instance.
+	 * @param	AutoDestroy		Whether to destroy this sound when it finishes playing.
+	 * 							Leave this value set to "false" if you want to re-use this FlxSound instance.
 	 * @param	AutoPlay		Whether to play the sound.
 	 * @param	URL				Load a sound from an external web resource instead.  Only used if EmbeddedSound = null.
+	 * @param	OnComplete		Called when the sound finished playing
+	 * @param	OnLoad			Called when the sound finished loading.  Only used if EmbeddedSound = null
```

Comment:
>Only used if EmbeddedSound = null

I guess that's debatable. It could be convenient to call it immediately in that case?

Reply:
I went back and forth on it. looking at it now, the method is called `load()` so I'm sure they would expect onLoad to fire for both cases. I'll add it